sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -85,13 +85,11 @@ repos:
     rev: v1.26.2
     hooks:
       - id: pyupgrade
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v1.9.4
-    hooks:
-      - id: seed-isort-config
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
+  - repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21-2
     hooks:
       - id: isort
         name: isort except __init__.py
```

Comment:
I wonder... Why do we ignore these files with isort? Don't we want sorted imports there too? :thinking: 

cc @sbidoul

Reply:
I think I made it like this because there are situations where the import order of models classes is important for the construction of the Odoo registry.

Reply:
I guess that was only back then in v7, where one had to instantiate models after declaration.

If there are isort exclusion tags, I'd vote for sorting these too. In corner cases, we just add those exclusion tags and go on.

Reply:
I remember at least one case where loading order of models matters (I think it's because defaults applied on fields), so I prefer to not automate this.